Modules/IndividualAssessment/classes/class.ilObjIndividualAssessmentGUI.php

```diff
@@ -381,6 +381,7 @@ protected function getEntryForStatus(int $status) : string
         }
     }
 
+    // PHP8-Review: Parameter's name changed during inheritance
```

Comment:
@katringross I might miss something, but to me the parameter in `ilObjectGUI` seems to be named the same.

Reply:
@klees Hmm I think you are right, we used PHPStan, maybe a false positive...